client/www/pages/docs/auth.md

```diff
@@ -28,3 +68,36 @@ Instant comes with support for auth. We currently offer [magic codes](/docs/auth
             /%}
 
 {% /nav-group %}
+
+## Low Level API
+
```

Comment:
I would add something like: 

"Sometimes you need finer control over auth states. In those cases you can use some of the lower-level API."